scripts/markdownlint-run.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-markdownlint . -p .github/markdown-lint/.markdownlintignore -c .github/markdown-lint/.markdownlint.yaml
+markdownlint . -p .github/markdown-lint/.markdownlintignore -c .github/markdown-lint/.markdownlint.yaml --fix
```

Comment:
let's leave it up to the caller - I did that here: https://github.com/Faire/yawn/pull/31